src/systemctl/systemctl.c

```diff
@@ -4594,7 +4594,7 @@ static int show_one(
         if (r < 0)
                 return log_error_errno(r, "Failed to map properties: %s", bus_error_message(&error, r));
 
-        if (streq_ptr(property_info.load_state, "not-found") && streq_ptr(property_info.active_state, "inactive"))
+        if (show_properties && streq_ptr(property_info.load_state, "not-found") && streq_ptr(property_info.active_state, "inactive"))
```

Comment:
The previous behavior was we were not able to print a error message and a error code when a property is not there and that case showing unit status was not affected. Now without the `show_properties`  we always showing the `property `  status in case of status also which should not happen.